test/params_api_test.c

```diff
@@ -469,8 +468,8 @@ static int test_param_signed_bignum(int n)
         goto err;
     if (!TEST_true(OSSL_PARAM_set_BN(&param, b)))
         goto err;
-    le_copy(buf2, len, bnbuf, sizeof(bnbuf));
-    if (!TEST_mem_eq(buf1, len, buf2, len))
+    le_copy(buf, len, bnbuf, sizeof(bnbuf));
```

Comment:
To keep the logic the same as in the test_param_bignum() I'd keep the first le_copy() call and remove this one instead.

Reply:
Although thinking about the logic it actually makes more sense to keep this le_copy() and change the other test to align with this one because we would then really compare the raw value with the result.

Reply:
Do you mean moving the `le_copy` in the `test_bignum` three lines down?

Reply:
Not just moving, but also changing what's le_copied so the bnbuf is le_copied instead of the raw data.

Reply:
And of course it would have to be moved in between the set_BN and TEST_mem_eq().

Reply:
Updated.  There were some more non-similarities that I changed in these functions.